resources/assets/sass/components/_travel-letter.scss

```diff
@@ -0,0 +1,34 @@
+.c-travel-letter {}
```

Comment:
What is travel-letter? Blog? Lets keep it consistent and rename blog to something better in one go.